src/Microsoft.DotNet.Darc/src/Darc/Operations/GatherDropOperation.cs

```diff
@@ -351,6 +351,7 @@ public string GetFileShareLocationForReleaseJson(DownloadedBuild build)
         { $"{githubRepoPrefix}dotnet/emsdk", (coreRepoCategory, "emsdk") },
         { $"{githubRepoPrefix}dotnet/sdk", (coreRepoCategory, "sdk") },
         { $"{githubRepoPrefix}dotnet/roslyn-analyzers", (coreRepoCategory, "roslyn-analyzers") },
+        { $"{githubRepoPrefix}dotnet/linker", (coreRepoCategory, "linker") },
```

Comment:
when I go to "[dotnet/linker (github.com)](https://github.com/dotnet/linker)" it says repo is archived, and it's being developed by "dotnet/runtime".